kibbeh/src/modules/layouts/MainLayout.tsx

```diff
@@ -112,17 +113,35 @@ export const MainLayout: React.FC<MainLayoutProps> = ({
 
   return (
     <>
-      <div className="fixed top-0 left-0 w-full z-10">{prepend}</div>
+      <ElectronHeader />
+      <div
+        className={`fixed left-0 w-full z-50`}
```

Comment:
```suggestion
        className={`fixed left-0 w-full z-10`}
```

Why did you add a higher z-index? It messes up the black thing that darkens the screen when you open the AccountOverlay. If it's an electron thing, make it electron exclusive

Reply:
done